tools/ci-scripts/codegen-diff/diff_lib.py

```diff
@@ -55,12 +55,12 @@ def generate_and_commit_generated_code(revision_sha, targets=None, preserve_aws_
     # Move generated code into codegen-diff/ directory
     get_cmd_output(f"rm -rf {OUTPUT_PATH}")
     get_cmd_output(f"mkdir {OUTPUT_PATH}")
-    if target_aws_sdk in targets:
-        # Compiling aws-config for semver checks baseline requires build artifacts to exist under aws/sdk/build
-        if preserve_aws_sdk_build:
-            get_cmd_output(f"cp -r aws/sdk/build/aws-sdk {OUTPUT_PATH}/")
-        else:
-            get_cmd_output(f"mv aws/sdk/build/aws-sdk {OUTPUT_PATH}/")
+    # if target_aws_sdk in targets:
```

Comment:
no longer needed or what?

Reply:
Both of the bits you commented on were early attempts to fix the doubled crates issue in the semver check, neither change actually made any difference so I can revert them

Reply:
Reverted in https://github.com/smithy-lang/smithy-rs/pull/3936/commits/5f0c533be2a6ac951b9084b6853a5bea6aa2baaa